android/src/main/java/com/swmansion/rnscreens/events/ScreenDismissedEvent.kt

```diff
@@ -1,20 +1,17 @@
 package com.swmansion.rnscreens.events
 
 import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableMap
 import com.facebook.react.uimanager.events.Event
-import com.facebook.react.uimanager.events.RCTEventEmitter
 
-class ScreenDismissedEvent(viewId: Int) : Event<ScreenDismissedEvent>(viewId) {
+class ScreenDismissedEvent(surfaceId: Int, viewId: Int) : Event<ScreenDismissedEvent>(surfaceId, viewId) {
     override fun getEventName() = EVENT_NAME
 
     // All events for a given view can be coalesced.
     override fun getCoalescingKey(): Short = 0
 
-    override fun dispatch(rctEventEmitter: RCTEventEmitter) {
-        val args = Arguments.createMap()
-        // on Android we always dismiss one screen at a time
-        args.putInt("dismissCount", 1)
-        rctEventEmitter.receiveEvent(viewTag, eventName, args)
```

Comment:
Are you sure we can just remove this code? Is it done like this in other libs/core ?

Reply:
Just take a look into [default implementation of this method](https://github.com/facebook/react-native/blob/e2ef6b98a061508bdd70a4c2747fe061c49b5c39/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/events/Event.java#L154-L168).

Reply:
Also I've just found [an example](https://github.com/facebook/react-native/commit/72d0ddc16f2f631003c3486e0a59e50c145ec613#diff-afc6f25f459dcf1e24a012e16862c869aa04b0de5795171efe14b7a88be08e64)